src/mpid/ch4/netmod/ofi/ofi_impl.h

```diff
@@ -38,6 +38,9 @@ ATTRIBUTE((unused));
 
 #define MPIDI_OFI_WIN(win)     ((win)->dev.netmod.ofi)
 
+#define MPIDI_OFI_NIC_NAME(nic)    (MPIDI_OFI_global.prov_use[nic]->domain_attr->name)
+#define MPIDI_OFI_DEFAULT_NIC_NAME (MPIDI_OFI_NIC_NAME(0))
```

Comment:
I worry that in the case we are using non-default nic, then we will be providing *misleading* information.

Reply:
I'll add a patch to use explicit info.

Reply:
As discussed, I've change the error text to state `default nic=%s`. In cases where the provider does not store nic information in the domain attributes, the user will see `default nic=(n/a)`.